### PR TITLE
BIOWDL-354: Update parameter_meta and TALON task cores.

### PR DESCRIPTION
**Pull request by JasperBoom**
### Checklist
- [x] Pull request details were added to CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,53 +11,55 @@ that users understand how the changes affect the new version.
 
 version 2.1.0-dev
 ---------------------------
-+ Updated TALON to version 4.4
++ Updated parameter_meta sections for Minimap2 and TranscriptClean.
++ Updated cores variable for TALON.
```
Comment (DavyCats) [inline CHANGELOG.md]: Updated how?

File: minimap2.wdl [hunk 2]
```diff
@@ -59,35 +59,55 @@ task Indexing {
...
         Int? maxIntronLength
+        Int? maxFragmentLength
         Boolean? skipSelfAndDualMappings
         Int? retainMaxSecondaryAlignments
         Int? matchingScore
         Int? mismatchPenalty
         String? howToFindGTAG
-        Boolean? secondaryAlignment
         Boolean? addMDtagToSAM
+        Boolean? secondaryAlignment
```
Comment (DavyCats) [inline minimap2.wdl]: These `Boolean?` inputs could also be `Boolean ... = false`

File: CHANGELOG.md (hunk 1 above)
Comment (JasperBoom) [reply CHANGELOG.md]: Updated to the new wdl-aid structure.

Comment (DavyCats) [reply CHANGELOG.md]: I meant the cores variable: `Updated cores variable for TALON.`
Looks like the default value changed, that should be mentioned specifically.

Comment (JasperBoom) [reply CHANGELOG.md]: Ah oke, I have addressed it in the CHANGELOG now.

**Pull request merged**